### PR TITLE
UPDATE: relocated files adn creating View and Edit members pages

### PR DESCRIPTION
**Pull request by ValentinNM**
UPDATE
Relocated files from _components_ to _pages_ **directory**
Added Router to `app.js`
Created `ViewMember` and `EditMember` members pages as well as added context
Refactored and cleaned up the `Header` component. 

Awaiting approval to merge.
thanks

### CODE REVIEW

File: src/Pages/CreateMembers.jsx [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
 export default function (props) {
   return (
     <section className="two-column-grid-forms">
       <div className="available-classes">
         <h2>Available Classes: </h2>
         <ul>
-          {props.map((oneClass, index) => {
-            const {} = props;
+          {/* {props.map((oneClass, index) => {
+            const {} = props; */}
```
Comment (Auenc) [inline src/Pages/CreateMembers.jsx:13]: Might be a good idea to remove this commented code if it's not needed anymore :)

File: src/Pages/EditMember.jsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import TextField from "@mui/material/TextField";
+
+export default function EditMember(props) {
+  return (
+    <section className="two-column-grid-forms">
+      <div className="available-classes">
+        <h2>Available Classes: </h2>
+        <ul>
+          {/* {props.map((oneClass, index) => {
+            const {} = props; */}
```
Comment (Auenc) [inline src/Pages/EditMember.jsx:13]: Same as above

File: src/components/ListOfMembers.jsx [hunk 3]
```diff
@@ -1,12 +1,20 @@
 import Button from '@mui/material/Button';
+import { Link } from 'react-router-dom';
 
 export default function ListOfMembers(props) {
+  
+  const handleMemberToView = (event, member) => { 
+    console.log("handleMemberToView: ", member)
```
Comment (Auenc) [inline src/components/ListOfMembers.jsx:7]: Consider removing the console log if it's no longer needed :)

**Review by Auenc — APPROVED** [review]
Looks good!

File: src/Pages/CreateMembers.jsx (hunk 1 above)
Comment (ValentinNM) [reply src/Pages/CreateMembers.jsx:13]: this was part of the file setup
will delete as going forward 
thanks

File: src/Pages/EditMember.jsx (hunk 2 above)
Comment (ValentinNM) [reply src/Pages/EditMember.jsx:13]: this was part of the file setup
will have to pass data through props at some point
thanks

File: src/components/ListOfMembers.jsx (hunk 3 above)
Comment (ValentinNM) [reply src/components/ListOfMembers.jsx:7]: will delete as going forward
thanks

**Pull request merged**